Extensions/permalink_copy.js

```diff
@@ -0,0 +1,124 @@
+//* TITLE permalink_copy **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION **//
+//* DEVELOPER New-XKit **//
```

Comment:
```suggestion
//* DEVELOPER new-xkit **//
```
consistency

Reply:
Too much copy-paste, will add that to something actually meaningful.